src/nonce-manager/utils.ts

```diff
@@ -0,0 +1,15 @@
+import Contract from 'web3-eth-contract'
+import Web3 from 'web3'
+
+export function encodeNonce(
+    contract: Contract.Contract,
+    address: string
+): string {
+    return contract.methods._nonces(address).encodeABI()
```

Comment:
Replace `_nonces` with `nonce`. And let's remove `limitOrderV3Contract: Contract.Contract`. This instance is not needed. Just use NONCE_SELECTOR = `0x70ae92d2` in constants and `address.substring(2).padStart(64, '0')`